src/pokemanki/gui/pokemanki_trade.py

```diff
@@ -0,0 +1,197 @@
+# Form implementation generated from reading ui file 'designer/pokemanki_trade.ui'
+#
+# Created by: PyQt6 UI code generator 6.4.0
+#
+# WARNING: Any manual changes made to this file will be lost when pyuic6 is
+# run again.  Do not edit this file unless you know what you are doing.
+
+import ctypes
+import platform
+
+from aqt import QDialog
+from aqt.webview import AnkiWebView
+from PyQt6 import QtCore, QtGui, QtWidgets
+
```

Comment:
Import the autogenerated code:
```suggestion

from .forms.qt6 import pokemanki_trades
```